beacon-chain/rpc/eth/validator/validator.go

```diff
@@ -37,6 +37,7 @@ import (
 )
 
 var errInvalidValIndex = errors.New("invalid validator index")
+var errParticipation = status.Errorf(codes.Internal, "Could not obtain epoch participation")
```

Comment:
There is no format here, prefer `status.Error`
```suggestion
var errParticipation = status.Error(codes.Internal, "Could not obtain epoch participation")
```